test/suites/container_devices_tpm.sh

```diff
@@ -1,28 +1,14 @@
 test_container_devices_tpm() {
   if ! modprobe tpm_vtpm_proxy; then
+    if [[ "$(uname -r)" =~ -azure$ ]]; then
```

Comment:
So this cant run on github?

Reply:
I improved the commit slicing around this but to reiterate, there is no way for us to test the TPM on GitHub... unless we can convince the kernel team to flip the `CONFIG_TCG_VTPM_PROXY` switch.